package.json

```diff
@@ -76,7 +76,8 @@
     "url-loader": "4.1.1",
     "winston": "3.3.3",
     "winston-mongodb": "5.0.5",
-    "yamljs": "0.3.0"
+    "yamljs": "0.3.0",
+    "csv-stringify": "6.3.0"
```

Comment:
Generally don't like to add new packages to ACAS when it can be avoided but in this case, I can't find any examples of usage of the other package we have called 'json2csv'. It also seems to be abandoned so it's worth using this one either way.